src/shared/components/palette/collection/write-collection-field.html

```diff
@@ -8,7 +8,6 @@ <h2 class="heading-h2">
     <button class="button" type="button" (click)="addItem(true)" [disabled]="isNotAuthorisedToCreate()">Add new</button>
     <h2 class="heading-h2 error-spacing">
       <span *ngIf="caseField.hint_text" class="form-hint">{{caseField.hint_text}}</span>
-      <span *ngIf="formArray.errors" class="error-message">{{formArray.errors | ccdFirstError}}</span>
```

Comment:
please explain in details why this change

Reply:
DONE change reverted.